template/8-user_analytics.md

```diff
@@ -3,6 +3,9 @@
 *Goal: understand how users are using the app.*
 
 *Which are the key metrics?*
+- Number of active users
+- Number of posts created
+- Number of comments
 
```

Comment:
We could also add the centauri point metric that we can follow for each users.

```suggestion
- Number of Centauri points per user.
```

Reply:
fixed in 66d57795bdc4e53806ac477fad63e2fe625c382f